environment.go

```diff
@@ -785,6 +789,25 @@ type BuildBaronProject struct {
 	BFSuggestionFeaturesURL string `mapstructure:"bf_suggestion_features_url" bson:"bf_suggestion_features_url"`
 }
 
+type AnnotationsSettings struct {
+	// a list of jira fields the user wants to display in addition to state assignee and priority
+	JiraCustomFields []JiraField `mapstructure:"jira_custom_fields" bson:"jira_custom_fields"`
+	// the endpoint that the user would like to send data to when the file ticket button is clicked
+	FileTicketWebHook WebHook `mapstructure:"web_hook" bson:"web_hook"`
+}
+
+type JiraField struct {
+	// the name that jira calls the field
+	Field string `mapstructure:"field" bson:"field"`
+	// the name the user would like to call it in the UI
+	DisplayText string `mapstructure:"display_text" bson:"display_text"`
+}
+
+type WebHook struct {
+	Endpoint string `mapstructure:"endpoint" bson:"endpoint"`
+	secret   string `mapstructure:"secret" bson:"secret"`
```

Comment:
is this lowercase intentional?

Reply:
yes, because we need to be very intentional about how we expose it

Reply:
but how will it be read by the resolver in the graphql package to send down to the UI? How will it be stored in the db (assuming the bson marshaler won't be able to read it)? How will the mapstructure decode it when we need to save it?

Reply:
good point :)